proto/alliance/gov.proto

```diff
@@ -19,23 +20,27 @@ message MsgCreateAllianceProposal {
     // The reward weight specifies the ratio of rewards that will be given to each alliance asset
     // It does not need to sum to 1. rate = weight / total_weight
     // Native asset is always assumed to have a weight of 1.
-    string reward_weight =4 [
+    string reward_weight = 4 [
         (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
         (gogoproto.nullable)   = false
     ];
+    // set a bound of weight range to limit how much reward weights can scale. 
+    RewardWeightRange reward_weight_range = 5 [
```

Comment:
Same comment regarding appending new fields instead of re-indexing everything else